apps/transport/client/javascripts/map.js

```diff
@@ -94,10 +94,14 @@ function displayRegions (map, featureFunction, style) {
 function addStaticPTMap (id, view) {
     const map = makeMapOnView(id, view)
 
+    const nbBaseSchedule = (feature) => {
```

Comment:
`const nbBaseSchedule = (feature) => feature.properties.dataset_types.pt`

Reply:
:+1: at start the function was more complex, sorry corrected